.github/workflows/e2e-tests.yml

```diff
@@ -13,7 +13,7 @@ on:
 jobs:
   e2e-tests:
     env:
-      UV_HTTP_TIMEOUT: 600
+      UV_HTTP_TIMEOUT: 1000
```

Comment:
I think there's flakiness despite the timeout... wondering if we should instead lower it and when it fails, we restart the job. Otherwise it can be a full 10+ minutes until we notice...